StringCalculator/StringCalculator.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Collections.Generic;
+
+namespace StringCalculator
+{
+    public class StringCalculator
+    {
+        public StringCalculator()
+        {
+
+        }
+
+        public int Add(string stringInput)
+        {
+            var total = 0;
+            if (String.IsNullOrEmpty(stringInput)) return total;
+
+            if (stringInput.Contains("//"))
```

Comment:
"//" can be extracted as a constant. That way if this constant is used in many places. you only need to change it once, when you need to